build/docker/config/dendrite.yaml

```diff
@@ -318,6 +318,17 @@ user_api:
     max_idle_conns: 2
     conn_max_lifetime: -1
 
+# Configuration for the Push Server API.
+push_server:
```

Comment:
i believe we do not need this block anymore after the changes of @neilalexander which make it part of **user_api**